src/framework/preferences/js/Panels.js

```diff
@@ -662,13 +673,25 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         },
         selectors: {
             textSize: ".flc-prefsEditor-min-text-size",
-            label: ".flc-prefsEditor-min-text-size-label",
+            label: ".flc-prefsEditor-min-text-size-label"
+        },
+        selectorsToIgnore: ["textSize"],
+        protoTree: {
+            label: {messagekey: "textSizeLabel"},
+            textSizeDescr: {messagekey: "textSizeDescr"},
+            smallIcon: {messagekey: "textSizeSmallIcon"},
+            largeIcon: {messagekey: "textSizeLargeIcon"},
+            multiplier: {messagekey: "multiplier"}
+        }
+    });
+
+    fluid.defaults("fluid.prefs.textSize.textfieldSlider", {
```

Comment:
1. This component doesn't have a base grade name, add a comment to explain the reason, or even add a base grade ```fluid.component``` for the refactoring mention in (2). The same comment applies to ```fluid.prefs.textSize.textfieldButtons```.
2. Again, due to the similarity, create a base grade component to be shared by ```fluid.prefs.textSize.textfieldSlider``` and ```fluid.prefs.lineSpace.textfieldSlider``` and another grade to be shared by ```fluid.prefs.textSize.textfieldButtons``` and ```fluid.prefs.lineSpace.textfieldButtons```.